### DISCUSSION TITLE
Error: Could not manually update certificate

### DISCUSSION THREAD

**Discussion by kiwidyne** [rcl-ssl/docs#33] [General]
I'm trying to renew a previously issued certificate and I am receiving the following error:

Error:Could not manually update certificate : [redacted] with challenge : HTTP for target Azure App Service. BadRequest : Failure in AzureService in AzureWebAppService : PostAsGet. Failed response : Unauthorized in AzureWebAppService : PostAsGet. {"error":{"code":"InvalidAuthenticationTokenTenant","message":"The access token is from the wrong issuer 'https://sts.windows.net/7c8b9ff2-6593-495c-b01f-cfe9fe7c7a67/'. It must match the tenant 'https://sts.windows.net/9738e45a-6725-40f0-86be-ca649859c229/' associated with this subscription. Please use the authority (URL) 'https://login.windows.net/9738e45a-6725-40f0-86be-ca649859c229' to get the token. Note, if the subscription is transferred to another tenant there is no impact to the services, but information about new tenant could take time to propagate (up to an hour). If you just transferred your subscription and see this error message, please try back later."}}. SSL Azure App Service Install HTTP Validation Record - RCL V1 API. If this error is persistent, please delete this certificate and re-create a new one.

What am I doing wrong here?

I am probably going to just delete it and re-issue a new certificate. I hope that isn't broken, too.

**Comment by rcladmin** [comment]
i ma looking into this.

**Reply by kiwidyne** [reply]
Thanks. I was able to create a new standalone cert and get it installed, so the crisis has passed, but I'd like to figure out what went wrong and what's wrong with my account with regards to the other issue I opened today.

**Reply by rcladmin** [reply]
It seems you are using an MSA account to sign in. Go to the subscription section under 'Work or School Account', what do you see? Do you have a work/school account registered?

https://docs.rclapp.com/authorization/sign-in-accounts.html
